code/__DEFINES/stack_trace.dm

```diff
@@ -0,0 +1,4 @@
+/// gives us the stack trace from CRASH() without ending the current proc.
+#define stack_trace(message) _stack_trace(message, __FILE__, __LINE__)
+
+#define WORKAROUND_IDENTIFIER "%//%"
```

Comment:
can you comment what this is for?
we have `SOME_MAGIC_VALUE` 

also personally we should consider

`/magic_marker/xyz` or something at this point because we have a _lot_ of unique value-types that are just .. strings, unless we need this to be a string for some reason?

Reply:
nevermind this **is** a string

what?
wack.

Put a comment saying what this is for please